types/artifacts.d.ts

```diff
@@ -569,6 +569,7 @@ declare global {
         speedIndexTs: number | undefined;
         estimatedInputLatency: number | undefined;
         estimatedInputLatencyTs: number | undefined;
+        maxPotentialFID: number | undefined;
```

Comment:
I missed this in #9814, but some of these aren't `| undefined` (the ones without the `=` in the old [`UberMetricsItem`](https://github.com/GoogleChrome/lighthouse/pull/9814/files#diff-5f0b57a6c12193c8f886ce0636c7ef07L134-L174)). It would be good to restore that at some point...we make everyone suffer if `FCP` is ever missing, when it succeeds we should reward everyone with tighter types since they're available :)